contracts/ClaimsReward.sol

```diff
@@ -222,12 +227,10 @@ contract ClaimsReward is Iupgradable {
      */
     function getAllPendingRewardOfUser(address _add) public view returns(uint total) {
         uint caReward = getRewardToBeDistributedByUser(_add);
-        uint commissionEarned = td.getStakerTotalEarnedStakeCommission(_add);
-        uint commissionReedmed = td.getStakerTotalReedmedStakeCommission(_add);
-        uint unlockableStakedTokens = tf.getStakerAllUnlockableStakedTokens(_add);
+        uint pooledStakingReward = pooledStaking.stakerReward(_add);
+        uint stakedTokens = pooledStaking.stakerStaked(_add);
         uint governanceReward = gv.getPendingReward(_add);
-        total = caReward.add(unlockableStakedTokens).add(commissionEarned.
-        sub(commissionReedmed)).add(governanceReward);
+        total = caReward.add(stakedTokens).add(pooledStakingReward).add(governanceReward);
```

Comment:
I think it's worth adding on this branch the updates from. I think this is the only place where it impact you, there is an extra line missing (`uint unlockableStakedTokens = tf.getStakerAllUnlockableStakedTokens(_add);`)
https://github.com/NexusMutual/smart-contract-upgrades/tree/fix-issue-181-unlockBug

Reply:
You can assume they will be on master when we deploy/merge this